redis-utils/src/main/java/org/ballerinalang/redis/actions/AbstractRedisAction.java

```diff
@@ -95,7 +96,7 @@ protected AbstractRedisAction() {
         }
     }
 
-    protected BInteger append(String key, String value, RedisDataSource<String, String> redisDataSource) {
+    protected static BInteger append(String key, String value, RedisDataSource<String, String> redisDataSource) {
```

Comment:
Shall we make these methods package private?

Reply:
These methods are accessed from the child classes (eg: "StringActions",  "KeyActions").

Reply:
Yes. But the  classes like`StringActions` , `KeyActions` are in the same package no? So cant we make the methods package private (i.e. `static BInteger append(String key, String value, RedisDataSource<String, String> redisDataSource) ` instead of `protected static BInteger append(String key, String value, RedisDataSource<String, String> redisDataSource) `

Reply:
Ack.

Reply:
Updated in e26d6d2